editor/src/core/shared/uid-utils.ts

```diff
@@ -29,6 +29,21 @@ import { UTOPIA_PATH_KEY } from '../model/utopia-constants'
 import { mapDropNulls } from './array-utils'
 import { ElementPath } from './project-file-types'
 
+export const MOCK_NEXT_GENERATED_UIDS: { current: Array<string> } = { current: [] }
+export const MOCK_NEXT_GENERATED_UIDS_IDX = { current: 0 }
+
+export function generateMockNextGeneratedUID(): string | null {
+  if (
+    MOCK_NEXT_GENERATED_UIDS.current.length > 0 &&
+    MOCK_NEXT_GENERATED_UIDS_IDX.current < MOCK_NEXT_GENERATED_UIDS.current.length
+  ) {
+    MOCK_NEXT_GENERATED_UIDS_IDX.current += 1
+    return MOCK_NEXT_GENERATED_UIDS.current[MOCK_NEXT_GENERATED_UIDS_IDX.current - 1]
```

Comment:
small nit: I think this would be more readable as e.g. this

```ts
const nextID = MOCK_NEXT_GENERATED_UIDS.current[MOCK_NEXT_GENERATED_ID.current]
MOCK_NEXT_GENERATED_ID.current += 1
return nextID
```

(of course there's also the option of `++`, but imho it'd not be an improvement as far as readability is concernet)

Reply:
I just transplanted the original code with no changes.

Reply:
oh yeah for sure, just noting the nit :)

Reply:
Fixed in c8df6e8c8.